librz/core/cmd_debug.c

```diff
@@ -832,29 +832,38 @@ static int step_until_inst(RzCore *core, const char *instr, bool regex) {
 	return true;
 }
 
-static int step_until_optype(RzCore *core, const char *_optypes) {
+static void dbg_follow_seek_register(RzCore *core) {
+	int follow = rz_config_get_i(core->config, "dbg.follow");
+	rz_cons_break_pop();
```

Comment:
move this outside this function, because it is done only in one place, not everywhere.

Reply:
Sure, alright. You are referring to `rz_cons_break_pop();`, right?

Reply:
yes!